build-scripts/hack/sync-registry.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+docker run -v "${GITHUB_WORKSPACE}/.github/data/sync-registry-config.yaml":/config.yaml quay.io/skopeo/stable:v1.15 sync \
```

Comment:
We do not maintain this docker image, and that can cause troubles in the future. We could use `skopeo` from the ubuntu archive instead.